tests/unit_tests/test_capi.py

```diff
@@ -419,3 +419,45 @@ def test_id_map(capi_init):
 
     ids = openmc.capi.plot.id_map(s)
     assert np.array_equal(expected_ids, ids)
+
+def test_property_map(capi_init):
+    expected_properties = np.array(
+        [[(293.6, 0.740582), (293.6, 6.55), (293.6, 0.740582)],
+         [ (293.6, 6.55), (293.6, 10.29769),  (293.6, 6.55)],
+         [(293.6, 0.740582), (293.6, 6.55), (293.6, 0.740582)]], dtype='float')
+
+    # create a plot object
+    s = openmc.capi.plot._PlotBase()
+    s.width = 1.26
+    s.height = 1.26
+    s.v_res = 3
+    s.h_res = 3
+    s.origin = (0.0, 0.0, 0.0)
+    s.basis = 'xy'
+    s.level = -1
+
+    properties = openmc.capi.plot.property_map(s)
+    assert np.allclose(expected_properties, properties, atol=1e-04)
+
+
+def test_position(capi_init):
+
+    pos = openmc.capi.plot._Position(1.0, 1.0, 1.0)
+
+    assert pos[0] == 1.0
+    assert pos[1] == 1.0
+    assert pos[2] == 1.0
```

Comment:
I'd suggest:
```Python
assert tuple(pos) == (1.0, 1.0, 1.0)
```
This also has the benefit that you're implicitly testing for the `IndexError` by virtue of iterating over `pos` when building the tuple, so you could remove the specific check below for raising `IndexError`.

Also, good idea to pick numbers that are different.